hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -1678,7 +1678,8 @@ public void testTrash() throws Exception {
     }, 1000, 120000);
 
     // userTrash path will contain the checkpoint folder
-    Assert.assertEquals(1, fs.listStatus(userTrash).length);
+    FileStatus[] statusList = fs.listStatus(userTrash);
+    Assert.assertTrue(1 <= statusList.length);
```

Comment:
you specifically are checking it isn't 0? listStatus can't return negative in any case.

Can we use ``assertNotEquals``? or if there is some other case possible can we put ``statusList.length`` in the message, earlier assertEquals would have printed statusList.length value in case of a failure

Reply:
Updated to use `assertNotEquals`, added `statusList` to assertion message.